packages/components/src/components/calendar/views/calendar-days.jsx

```diff
@@ -171,4 +174,4 @@ Days.propTypes = {
     disabled_days: PropTypes.arrayOf(PropTypes.number),
 };
 
-export default Days;
+export default React.memo(Days);
```

Comment:
What is the reason behind using `React.memo`?

Reply:
@matin-binary, to optimize the render performance

Reply:
I don't think it has much of effect on the performance plus props are getting ignored here which is not correct.

Reply:
I have deleted React.memo